src/map.cpp

```diff
@@ -4580,11 +4580,11 @@ void map::apply_in_fridge( item &it, int temp )
 {
     unsigned int diff_freeze = abs(temp - FREEZING_TEMPERATURE);
     diff_freeze = diff_freeze < 1 ? 1 : diff_freeze;
-    diff_freeze = diff_freeze > 10 ? 10 : diff_freeze;
+    diff_freeze = diff_freeze > 5 ? 5 : diff_freeze;
```

Comment:
You can use `std::min( 5, diff_freeze )` for this and `std::max( 1, diff_freeze )` above. The same for `diff_cold` below.

Reply:
would it be better performance wise?

Reply:
Honestly, I don't know. For me it's just more clear and readable. Need consulting from more competent guys.

Reply:
> would it be better performance wise?

It won't be more slow, but much more readable.